__tests__/select.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {mount} from 'enzyme';
+
+import Component from '../src/select';
```

Comment:
I see how this works, but I'd prefer to see the real component name used.
`Select` instead of `Component`.